cli/helpers/process.go

```
@@ -0,0 +1,63 @@
+package helpers
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"runtime"
+	"syscall"
+
+	"github.com/jensneuse/abstractlogger"
+
+	"github.com/wundergraph/wundergraph/pkg/loadvariable"
+	"github.com/wundergraph/wundergraph/pkg/wgpb"
+)
+
+func ServerPortFromConfig(configJsonPath string) (int, error) {
+	data, _ := ioutil.ReadFile(configJsonPath)
```

`ioutil` is deprecated, now it's `os.ReadFile`. Why are you ignoring an error? `data` could be `nil` and you will get a weird error: `unexpected end of JSON input` (line 25)